assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -272,10 +276,6 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi, encodeMarkup }) => {
 
         const preppedStoryProps = reshapeStoryPreview(storyProps);
 
-        const getStoryMarkup = await import(
-          /* webpackChunkName: "chunk-getStoryMarkup" */ '../../../edit-story/output/utils/getStoryMarkup'
-        );
-
```

Comment:
This prevented the markup from ever being returned / inhibited rendering the preview. Maybe I'm missing something?

Reply:
Ahh, yeaup. I think it's the default module here so either:
```
const { default: getStoryMarkup } = await import(
  /* webpackChunkName: "chunk-getStoryMarkup" */ '../../../edit-story/output/utils/getStoryMarkup'
);
const markup = await getStoryMarkup(...)
```
or this (which is the approach we take above)
```
const getStoryMarkup = await import(
  /* webpackChunkName: "chunk-getStoryMarkup" */ '../../../edit-story/output/utils/getStoryMarkup'
);
const markup = await getStoryMarkup.default(...)
```

should allow the dynamic import to work here

Reply:
❤️  ok thank you that fixed it!  All better.